Pocket/pocket-ios
# SwiftUI Home - Navigation

## Goal
* This PR adds navigation to the new SwiftUI based Home screen

## Test Steps
* Build/run 
* Go to SwiftUI home and make sure that tapping on elements or on "See all" buttons produces the same effect as the existing home

> [!NOTE]
> some of the detail views triggered by the navigation will contain placeholders, at this time

Related Issues
POCKET-10145
POCKET-10146
POCKET-10620
POCKET-10621
POCKET-10622
POCKET-10623

## Screenshots
